INSTALL.md

```diff
@@ -26,133 +28,115 @@ hardware) by running `qvm-start sys-usb`.
 5. Update dom0 and install `make`, referring again to
 [the next section of SDW docs](https://workstation.securedrop.org/en/stable/admin/install.html#apply-dom0-updates-estimated-wait-time-15-30-minutes)
 
+In our case, we also install `open-vm-tools` and run `sudo systemctl enable vmtoolsd`,
+as our scripts use vmtoolsd to issue commands to the dom0 from the VMware API.
+
 ```
-sudo qubes-dom0-update make
+sudo qubes-dom0-update make open-vm-tools
 ```
 
 6. Run any updates you see in the Qubes menu and then reboot.
 
-7. In dom0, create the sd-ssh StandaloneVM:
+7. In dom0, create the sd-dev StandaloneVM:
 
 ```
-sudo qvm-create --standalone --template fedora-37 --label red sd-ssh
-qvm-volume resize sd-ssh:root 50G
-qvm-volume resize sd-ssh:private 20G
-qvm-tags sd-ssh add sd-client
+sudo qvm-create --standalone --template fedora-37 --label red sd-dev
```

Comment:
Bump to fedora-38 (which SDW uses currently) since fedora-37 is EOL?

Reply:
Ah. I think I did use the fedora-38-xfce template on the Qubes 4.2 VMs, but the 4.1 ones might still be fedora-37, since out-of-the-box install of Qubes 4.1 you don't have a fedora 38 template (and I was mindful not to customize too much after install).

I'll look at replacing it on 4.1 with fedora-38 (and re-snapshot)